src/main/java/editCard/EditCardOutputData.java

```diff
@@ -1,27 +1,28 @@
-package editCard;/*
+/*
  * Author: Albert Yan
  * Date: October 31/22
  * The output data class for edit card systems, packages data related to output together to be sent to viewmodel
  */
 
```

Comment:
see comment in the EditCardController class

Reply:
No way youre calling out every comment from the edit card use case, but Done.